access/resource_sql_permissions.go

```diff
@@ -50,7 +50,7 @@ func (ta *SqlPermissions) typeAndKey() (string, string) {
 		return "VIEW", fmt.Sprintf("`%s`.`%s`", ta.actualDatabase(), ta.View)
 	}
 	if ta.Database != "" {
-		return "DATABASE", fmt.Sprintf("`%s`", ta.Database)
+		return "DATABASE", ta.Database
```

Comment:
backticks are valid quoting characters. why do you remove them?

Reply:
Hi !  I explain it there https://github.com/databricks/terraform-provider-databricks/issues/1764#issuecomment-1363029378 with logs and an example 
privilege_assignments are planned and applied every times because of this line 
I didn't find the root cause